string robot_tool_surface				# name of the robot tool surface that calibration values should be updated for
---
bool success                            
float64 calibration_x					# the tool x value to update the urdf to
float64 calibration_y					# the tool y value to update the urdf to
float64 calibration_z					# the tool z value to update the urdf to
float64 calibration_roll				# the tool roll value to update the urdf to
float64 calibration_pitch				# the tool pitch value to update the urdf to
float64 calibration_yaw				    # the tool yaw value to update the urdf to
